contracts/minters/open-edition-minter/src/contract.rs

```diff
@@ -313,16 +313,36 @@ fn _execute_mint(
         factory_params.mint_fee_bps.bps_to_decimal()
     };
     let network_fee = mint_price.amount * mint_fee;
+
     // This is for the network fee msg
-    checked_fair_burn(
-        &info,
-        network_fee.u128(),
-        Some(
-            deps.api
-                .addr_validate(&factory_params.extension.dev_fee_address)?,
-        ),
-        &mut res,
-    )?;
+    // send non-native fees to community pool
+    if mint_price.denom != NATIVE_DENOM {
+        // only send non-zero amounts
+        // send portion to dev addr
+        if !network_fee.is_zero() {
+            ibc_denom_fair_burn(
+                coin(network_fee.u128(), mint_price.denom.to_string()),
+                Some(
+                    deps.api
+                        .addr_validate(&factory_params.extension.dev_fee_address)?,
+                ),
+                &mut res,
+            )?;
+            // let msg =
+            //     create_fund_community_pool_msg(coins(network_fee.u128(), mint_price.clone().denom));
+            // res = res.add_message(msg);
+        }
+    } else {
+        checked_fair_burn(
```

Comment:
i guess this branch should also have the network fee zero check?

Reply:
yea you're right, should check 0fee